lib/src/main/java/org/team100/lib/sensors/Team100CANIMU.java

```diff
@@ -0,0 +1,88 @@
+package frc.robot;
+
+import edu.wpi.first.hal.CANData;
+import edu.wpi.first.util.sendable.Sendable;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj.CAN;
+
+public class Team100CANIMU {
+    public static final int CAN_DEVICE_MANUFACTURER = 8;
+    public static final int CAN_DEVICE_TYPE = 4;
+    public static final int API_ID_EULER_ANGLES = 17;
+    public static final int API_ID_QUATERNION = 18;
+    // the 16 bit values on the CAN bus must be divided by 163.84 then 200 should be
+    // subtracted from the value
+    public static final double INT_TO_ENG_COEFF = 1 / 163.84;
+    public static final double INT_TO_ENG_OFFSET = -200;
+    public static final double INT_TO_ENG_ACC = 1 / 655.36;
+
+    private long rollValueInt, pitchValueInt, yawValueInt, accValueInt;
+    private int deviceID;
+    private boolean result;
```

Comment:
it doesn't look like "result" is ever actually used here (i.e. it is only ever written to).  did you mean to use it for something?